README.adoc

```diff
@@ -314,18 +314,18 @@ pom.xml
 include::finish/pom.xml[]
 ----
 
-For your test classes to have access to JSON-B, the [hotspot=Yasson file=1]`yasson` dependency has been added in your `pom.xml`.
+For your test classes to have access to JSON-B, the [hotspot=Yasson file=1]`yasson` dependency has been added in your [hotspot file=1]`pom.xml`.
```

Comment:
Change
"For your test classes to have access to JSON-B, the [hotspot=Yasson file=1]`yasson` dependency has been added in your [hotspot file=1]`pom.xml`."
To
"The [hotspot=Yasson file=1]`yasson` dependency was added in your [hotspot file=1]`pom.xml` file so that your test classes have access to JSON-B."